plugin/src/main/java/org/opensearch/ml/action/controller/CreateControllerTransportAction.java

```diff
@@ -186,7 +186,8 @@ private void indexAndCreateController(
                     String errorMessage = getErrorMessage("Model controller saved into index, result:{}", modelId, isHidden);
                     log.info(errorMessage, indexResponse.getResult());
                     if (indexResponse.getResult() == DocWriteResponse.Result.CREATED) {
-                        mlModelManager.updateModel(modelId, isHidden, Map.of(MLModel.IS_CONTROLLER_ENABLED_FIELD, true));
+                        mlModelManager
+                            .updateModel(modelId, mlModel.getTenantId(), isHidden, Map.of(MLModel.IS_CONTROLLER_ENABLED_FIELD, true));
```

Comment:
Better to not add tenant ID to controller action. We are now adding multi tenancy to create controller but not to delete controller.